docs/src/literate-tutorials/hyperelasticity.jl

```diff
@@ -99,60 +99,51 @@ using Ferrite, Tensors, TimerOutputs, ProgressMeter, IterativeSolvers
 # ```math
 # \begin{align*}
 # \mathbf{P} &= \mathbf{F} \cdot \mathbf{S},\\
-# \frac{\partial \mathbf{P}}{\partial \mathbf{F}} &= \mathbf{I} \bar{\otimes} \mathbf{S} + 2\, \mathbf{F} \bar{\otimes} \mathbf{I} :
+# \frac{\partial \mathbf{P}}{\partial \mathbf{F}} &= \mathbf{I} \bar{\otimes} \mathbf{S} + 2\, \mathbf{F} \cdot
 # \frac{\partial \mathbf{S}}{\partial \mathbf{C}} : \mathbf{F}^\mathrm{T} \bar{\otimes} \mathbf{I}.
 # \end{align*}
 # ```
 
-#md # ```@raw html
-#md # <details class="admonition collapsible">
-#md # <summary class="admonition-header">
-#md # Derivation of <span>$\partial \mathbf{P} / \partial \mathbf{F}$</span>
-#md # </summary>
-#md # <div class="admonition-body">
-#md # ```
-#nb # ### Derivation of ``\partial \mathbf{P} / \partial \mathbf{F}``
-# Using the product rule, the chain rule, and the relations ``\mathbf{P} = \mathbf{F} \cdot
-# \mathbf{S}`` and ``\mathbf{C} = \mathbf{F}^\mathrm{T} \cdot \mathbf{F}``, we obtain the
-# following:
-# ```math
-# \begin{aligned}
-# \frac{\partial \mathbf{P}}{\partial \mathbf{F}} &=
-# \frac{\partial P_{ij}}{\partial F_{kl}} \\ &=
-# \frac{\partial (F_{im}S_{mj})}{\partial F_{kl}} \\ &=
-# \frac{\partial F_{im}}{\partial F_{kl}}S_{mj} +
-# F_{im}\frac{\partial S_{mj}}{\partial F_{kl}} \\ &=
-# \delta_{ik}\delta_{ml} S_{mj} +
-# F_{im}\frac{\partial S_{mj}}{\partial C_{no}}\frac{\partial C_{no}}{\partial F_{kl}} \\ &=
-# \delta_{ik}S_{lj} +
-# F_{im}\frac{\partial S_{mj}}{\partial C_{no}}
-# \frac{\partial (F^\mathrm{T}_{np}F_{po})}{\partial F_{kl}} \\ &=
-# \delta_{ik}S^\mathrm{T}_{jl} +
-# F_{im}\delta_{jq}\frac{\partial S_{mq}}{\partial C_{no}}
-# \left(
-# \frac{\partial F^\mathrm{T}_{np}}{\partial F_{kl}}F_{po} +
-# F^\mathrm{T}_{np}\frac{\partial F_{po}}{\partial F_{kl}}
-# \right) \\ &=
-# \delta_{ik}S_{jl} +
-# F_{im}\delta_{jq}\frac{\partial S_{mq}}{\partial C_{no}}
-# (\delta_{nl} \delta_{pk} F_{po} + F^\mathrm{T}_{np}\delta_{pk} \delta_{ol}) \\ &=
-# \delta_{ik}S_{lj} +
-# F_{im}\delta_{jq}\frac{\partial S_{mq}}{\partial C_{no}}
-# (F^\mathrm{T}_{ok} \delta_{nl} + F^\mathrm{T}_{nk} \delta_{ol}) \\ &=
-# \delta_{ik}S_{jl} +
-# 2\, F_{im}\delta_{jq} \frac{\partial S_{mq}}{\partial C_{no}}
-# F^\mathrm{T}_{nk} \delta_{ol} \\ &=
-# \mathbf{I}\bar{\otimes}\mathbf{S} +
-# 2\, \mathbf{F}\bar{\otimes}\mathbf{I} : \frac{\partial \mathbf{S}}{\partial \mathbf{C}}
-# : \mathbf{F}^\mathrm{T} \bar{\otimes} \mathbf{I},
-# \end{aligned}
-# ```
-# where we used the fact that ``\mathbf{S}`` is symmetric (``S_{lj} = S_{jl}``) and that
-# ``\frac{\partial \mathbf{S}}{\partial \mathbf{C}}`` is *minor* symmetric (``\frac{\partial
-# S_{mq}}{\partial C_{no}} = \frac{\partial S_{mq}}{\partial C_{on}}``).
-#md # ```@raw html
-#md # </div></details>
-#md # ```
+# !!! details "Derivation of $\partial \mathbf{P} / \partial \mathbf{F}$"
```

Comment:
This will only work for Documenters HTML output, not the notebooks (xref https://fredrikekre.github.io/Literate.jl/v2/tips/#admonitions-compatibility)

Reply:
Oh, is that the same above from #846 ? 
Or is it just the math in the header?

Reply:
Yea, see https://nbviewer.org/github/Ferrite-FEM/Ferrite.jl/blob/gh-pages/dev/tutorials/hyperelasticity.ipynb (forgot about this while reviewing that PR)

Reply:
Yes, realized too (also that the link in the docs hasn't been updated since the "example" time...)
What do you think about introducing the preprocessing commands suggested in the Literate docs?
Would be quite nice to be able to use admonitions in the examples.

Reply:
The citation is also broken x.x

Reply:
Should it be OK now after https://github.com/fredrikekre/Literate.jl/pull/259?

Reply:
I would just merge and look. Should be fine I think.

Reply:
Can we also quickly leave some links to material with an ELI5 for index notation and the used symbols here?

Reply:
Added link to my [website](https://knutam.github.io/tensors/Theory/IndexNotation/), hope that is ok?

Reply:
Well, the line break broke the notebook view, but looks correct in markdown. Bug in Literate @fredrikekre ? 



<details>

```
{
   "cell_type": "markdown",
   "source": [
    "> **Derivation of $\\partial \\mathbf{P} / \\partial \\mathbf{F}$**\n",
    ">\n",
    "> *Tip:* See [knutam.github.io/tensors](https://knutam.github.io/tensors/Theory/IndexNotation/) for\n",
    "> an explanation of the index notation used in this derivation\n",
    "\n",
    "    Using the product rule, the chain rule, and the relations $\\mathbf{P} = \\mathbf{F} \\cdot\n",
    "    \\mathbf{S}$ and $\\mathbf{C} = \\mathbf{F}^\\mathrm{T} \\cdot \\mathbf{F}$, we obtain the\n",
    "    following:\n",
    "    $$\n",
    "    \\begin{aligned}\n",
    "    \\frac{\\partial P_{ij}}{\\partial F_{kl}} &=\n",
    "    \\frac{\\partial (F_{im}S_{mj})}{\\partial F_{kl}} \\\\ &=\n",
    "    \\frac{\\partial F_{im}}{\\partial F_{kl}}S_{mj} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial F_{kl}} \\\\ &=\n",
    "    \\delta_{ik}\\delta_{ml} S_{mj} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\\frac{\\partial C_{no}}{\\partial F_{kl}} \\\\ &=\n",
    "    \\delta_{ik}S_{lj} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    \\frac{\\partial (F^\\mathrm{T}_{np}F_{po})}{\\partial F_{kl}} \\\\ &=\n",
    "    \\delta_{ik}S^\\mathrm{T}_{jl} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    \\left(\n",
    "    \\frac{\\partial F^\\mathrm{T}_{np}}{\\partial F_{kl}}F_{po} +\n",
    "    F^\\mathrm{T}_{np}\\frac{\\partial F_{po}}{\\partial F_{kl}}\n",
    "    \\right) \\\\ &=\n",
    "    \\delta_{ik}S_{jl} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    (\\delta_{nl} \\delta_{pk} F_{po} + F^\\mathrm{T}_{np}\\delta_{pk} \\delta_{ol}) \\\\ &=\n",
    "    \\delta_{ik}S_{lj} +\n",
    "    F_{im}\\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    (F^\\mathrm{T}_{ok} \\delta_{nl} + F^\\mathrm{T}_{nk} \\delta_{ol}) \\\\ &=\n",
    "    \\delta_{ik}S_{jl} +\n",
    "    2\\, F_{im} \\frac{\\partial S_{mj}}{\\partial C_{no}}\n",
    "    F^\\mathrm{T}_{nk} \\delta_{ol} \\\\\n",
    "    \\frac{\\partial \\mathbf{P}}{\\partial \\mathbf{F}} &=\n",
    "    \\mathbf{I}\\bar{\\otimes}\\mathbf{S} +\n",
    "    2\\, \\mathbf{F} \\cdot \\frac{\\partial \\mathbf{S}}{\\partial \\mathbf{C}}\n",
    "    : \\mathbf{F}^\\mathrm{T} \\bar{\\otimes} \\mathbf{I},\n",
    "    \\end{aligned}\n",
    "    $$\n",
    "    where we used the fact that $\\mathbf{S}$ is symmetric ($S_{lj} = S_{jl}$) and that\n",
    "    $\\frac{\\partial \\mathbf{S}}{\\partial \\mathbf{C}}$ is *minor* symmetric ($\\frac{\\partial\n",
    "    S_{mj}}{\\partial C_{no}} = \\frac{\\partial S_{mj}}{\\partial C_{on}}$)."
   ],
   "metadata": {}
  },
```

</details>

Reply:
But should be fixed for now by adding the workaround to remove the line break from the notebook.